sale_timesheet_nonbillable/views/account_analytic_line.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <!--
+        Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+        License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+    -->
+    <!-- <record id="account_analytic_line_tree" model="ir.ui.view">
```

Comment:
@alexey-pelykh IMO remove dead code, If not need

Reply:
Absolutely! In couple of hours as soon as this goes out of WIP stage, it will be lean and mean 👍